crates/sui-core/src/authority/authority_store_tables.rs

```diff
@@ -280,6 +280,16 @@ impl AuthorityPerpetualTables {
                     digest_prefix.clone(),
                 ),
             ),
+            (
+                "events".to_string(),
```

Comment:
why is this required?

Reply:
I just wanted to ask you :)
Without this line validator panics when opening authority store: (I am still deploying to confirm if panic goes away when table is returned)
```
thread 'sui-node-runtime' panicked at crates/sui-core/src/authority/authority_store_tables.rs:52:10
no entry found for key
```

Reply:
oh ok thanks, I'll take a look tomorrow. Is it rebased on top of main and is database empty?

Reply:
Yes to both

Reply:
Thanks. I confirmed that removing this line fixed panic